src/common/components/repository-row/index.js

```diff
@@ -82,7 +84,7 @@ class RepositoryRow extends Component {
 
     let caption;
     if (registerNameStatus.error) {
-      caption = <div>{ registerNameStatus.error.json.detail }</div>;
+      caption = <div>{ registerNameStatus.error.message }</div>;
```

Comment:
It would be nice to give it some error look, but it's not something we need to do in this PR.